Source/BugsnagConfiguration.h

```diff
@@ -142,16 +138,46 @@ BugsnagBreadcrumbs *breadcrumbs;
 @property BOOL autoNotify;
 
 /**
- * Determines whether app sessions should be tracked automatically. By default this value is false.
+ * Determines whether app sessions should be tracked automatically. By default this value is true.
  */
 @property BOOL shouldAutoCaptureSessions;
 
 /**
- * Set the endpoint to which tracked sessions reports are sent. This defaults to https://sessions.bugsnag.com,
- * but should be overridden if you are using Bugsnag On-premise, to point to your own Bugsnag endpoint.
+ * Retrieves the endpoint used to notify Bugsnag of errors
+ *
+ * NOTE: it is strongly recommended that you set this value via setEndpointsForNotify:sessions: instead.
+ *
+ * @see setEndpointsForNotify:sessions:
+ */
+@property(readwrite, retain, nullable) NSURL *notifyURL;
+
+/**
+ * Retrieves the endpoint used to send tracked sessions to Bugsnag
+ *
+ * NOTE: it is strongly recommended that you set this value via setEndpointsForNotify:sessions: instead.
```

Comment:
is it worth using something like [this](https://stackoverflow.com/a/3908766/549238) to make this as deprecated?